src/logging/mod.rs

```diff
@@ -53,7 +55,14 @@ impl KV for RequestMozlogFields {
         }
         serializer.emit_str("path", &self.path)?;
         serializer.emit_str("method", self.method)?;
+        Ok(())
+    }
+}
 
+impl Value for Settings {
```

Comment:
Would a more logical place for this be inside `src/settings/mod.rs`?

Reply:
I wasn't sure about that, I left it in the logging because it's something for logging and this saves me some import lines. I don't have a strong opinion about this, though. If you think it's best I can change it :)